core/schema-management.md

```diff
@@ -0,0 +1,40 @@
+# Schema Management
+
+## Problem Description
+
+To enable data exchange between business parties involved in a Baseline Workflow (or only Workstep?) (in short, parties), there must be present a set of pre-agreed schemas. 
+
+Furthermore, at least identifying different versions of the same schema must be possible for each Baseline instance. A Baseline instance may support different versions of the same schema.
+
+## Schema
+  
+Minimum data model describing the structure of exchanged messages between different components and APIs. It must include list of mandatory fields/properties and their types (and their description?).
+
+## Agreement on Schema
+(in scope?)
+An agreement could be _explicit_ with stating the list of agreed upon versioned schemas (e.g. structured as a Merkle Tree with schemas as leaves) by all the parties or _implicit_ by their implementation. 
```

Comment:
This is the basis for an interesting discussion worth engaging in.

Reply:
My current position is that we need to foresee/expect both. But will it be in scope of standard or not is another question.